README.md

```diff
@@ -1 +1,63 @@
-# Contact-Form-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portofolio
+
+
+> A portofolio webpage that displays all the projects I've been working on recently. 
+
+![screenshot]()
+
+
+
+## Live Demo
+Use the link below to access github page for my portofolio.
+
+- [Github_Page](https://zhadier.github.io/Contact-Form-Reviewable/)
+
+
+## Built With
+
+- HTML
+- CSS
+- VSCODE
+
+
+## Getting Started
+#### Step 1:
+
+- Clone this repo locally using git in the command line and the following command
+
+  git clone https://github.com/zhadier/Contact-Form-Reviewable
+  
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+#### Step 2:
+Copy All Project files and run using VSCODE. HTML can be viewed in browser.
+
+
+### Prerequisites
+- VSCODE
+
+
+### Usage
+- For display for potential Hirers.
+
+
+## Authors
+
```

Comment:
## Changes Required: 

⚠️ This is the most important part of your README. You should add: 
- Your GitHub
- Your LinkedIn
- Your Twitter.
This will allow people who enjoy your work to find ways to contact you! ⭐